e2e_samples/parking_sensors_synapse/scripts/deploy_infrastructure.sh

```diff
@@ -212,6 +212,8 @@ LOG_ANALYTICS_WS_ID=$loganalytics_id \
 LOG_ANALYTICS_WS_KEY=$loganalytics_key \
 KEYVAULT_NAME=$kv_name \
 AZURE_STORAGE_ACCOUNT=$azure_storage_account \
+SYNAPSE_SQL_PASSWORD=$SYNAPSE_SQL_PASSWORD \
+synapseAnalyticsSQLAdmin=$synapse_sqlpool_admin_username \
```

Comment:
Could we change to all caps and underscores (ei. SYNAPSE_ANALYTICS_SQL_ADMIN) to follow convention of passing external env variables?